setup.py

```diff
@@ -93,9 +87,17 @@
             'requests != 2.13.0',
         ],
         'test': [
-            'tox',
+            'boto >= 2.0b4',       # for qa/tasks/radosgw_*.py
+            'cryptography >= 2.7',  # for qa/tasks/mgr/dashboard/test_rgw.py
+            'nose', # for qa/tasks/rgw_multisite_tests.py',
             'pip-tools',
             'pytest',           # for tox.ini
+            'requests',         # for qa/tasks/mgr/dashboard/helper.py
+            'tox',
```

Comment:
Really tox? Isn't that a pre-requirement to run the tests inside tox?

Reply:
yeah, that's why i think it belongs to "test".

Reply:
I think it doesn't belong anywhere imo. It's just an external tool that you can use with `tox.ini`

Reply:
i see. i think we added it here just because it's a convenient way to add stuff to `requirements.txt`. the same applies to manhole, which is for debugging a task.

Reply:
I saw sometimes issues with system tox :-) so maybe it is a good thing to have it here, however I'm not 100% sure about that

Reply:
it's not useful (imo it's wrong). You already need tox to install requirements.txt in a venv via setuptools/distutils.
And `manhole` is totally different because `manhole` gets imported (`teuthology/__init__.py:import manhole`) so it needs to be in requirements.txt.
`tox` is not imported (or called via eg. subprocess) so it must not be in requirements.txt

Reply:
> it's not useful (imo it's wrong). You already need tox to install requirements.txt in a venv via setuptools/distutils.
it is not true, you do not need `tox`, you need `pip` to setup virtualenv or whatever what I usually do is:
```
virtualenv v ; . v/bin/activate
pip install -r requirements.txt
```

Reply:
gentlemen, please note, tox is not added in this change. i was merely reordering it. and i agree with @kshtsk, one don't need to tox to install packages specified requirements.txt.

Reply:
yeah. I also agree with that. But when you do `pip install -r requirements.txt`, `tox` will be installed which is useless because you already have your venv...

Reply:
anyway. Closing the discussion. Let's create a PR for removing tox if wanted and discuss there..